poetry/console/commands/shell.py

```diff
@@ -0,0 +1,24 @@
+from .venv_command import VenvCommand
+
+
+class ShellCommand(VenvCommand):
+    """
+    Spawns a shell within the virtual environment.
+
+    shell [options]
+    """
+
+    help = """The <info>shell</info> command spawns a shell, according to the
+<comment>$SHELL</> environment variable, within the virtual environment.
+If one doesn't exist yet, it will be created.
+"""
+
+    def handle(self):
+        if self.venv.is_venv():
+            self.line(f"Spawning shell within <info>{self.venv.venv}</info>")
```

Comment:
Poetry runs on Python versions below 3.6. This should probably be `"Spawning shell within <info>{venv}</info>".format(venv=self.venv.venv)`

Reply:
Done. I have always tend to neglect Python 2, so thanks for the heads up.

Reply:
f-strings doesn't work with python3 versions under 3.6, not only python 2